ui/src/pages/quoteRequests/QuoteRequests.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React from "react";
+import Contracts from "../../components/Contracts/Contracts";
+import { useStreamQuery, useParty, useLedger } from "@daml/react";
+import { QuoteRequest }
+  from "@daml.js/supplychain-1.0.0/lib/DA/RefApps/SupplyChain/QuoteRequest";
+import { useState } from "react";
+import { CreateEvent } from "@daml/ledger";
+import { OrderedProductList } from "./OrderedProductList";
+
+export default function QuoteRequests() {
+
+  const party = useParty();
+  const ledger = useLedger();
+  const roles =
+    useStreamQuery(QuoteRequest);
```

Comment:
Extremely minor, but triggered my OCD. :) Ignore, if this is done by formatter.
```suggestion
  const roles = useStreamQuery(QuoteRequest);
```